packages/twenty-server/src/engine/api/graphql/workspace-query-builder/factories/update-one-query.factory.ts

```diff
@@ -35,6 +35,7 @@ export class UpdateOneQueryFactory {
 
     const argsData = {
       ...computedArgs.data,
+      id: undefined, // do not allow updating an existing object's id
```

Comment:
Good point 👍 
nit: I guess we should throw a validationError but I don't know if the current FE sends the id so it might break it